src/WebJobs.Script/StorageProvider/StorageClientProvider.cs

```diff
@@ -177,12 +178,12 @@ protected virtual bool TryGetServiceUri(IConfiguration configuration, out Uri se
 
                 string accountName;
                 string uriStr;
-                if ((accountName = configuration.GetValue<string>("accountName")) != null)
+                if ((accountName = configuration?.GetValue<string>("accountName")) != null)
                 {
                     serviceUri = FormatServiceUri(accountName);
                     return true;
                 }
-                else if ((uriStr = configuration.GetValue<string>(serviceUriConfig)) != null)
+                else if ((uriStr = configuration?.GetValue<string>(serviceUriConfig)) != null)
                 {
                     serviceUri = new Uri(uriStr);
                     return true;
```

Comment:
Should we have a debug log here to indicate when we were unable to resolve the service URI?

Reply:
We try to resolve the serviceUri every time we create a client so adding a log here would add a good bit of noise. If we should be able to resolve a serviceUri, but can't in this method, we'd run into an exception in Microsoft.Extensions.Azure which would be caught and logged in this file in the TryGet() method.